src/Bridge/Symfony/DependencyInjection/SonataExporterExtension.php

```diff
@@ -30,24 +30,12 @@ public function load(array $configs, ContainerBuilder $container): void
         $configuration = new Configuration();
         $config = $processor->processConfiguration($configuration, $configs);
 
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
-        $loader->load('services.xml');
+        $loader = new PhpFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader->load('services.php');
 
-        $this->configureExporter($container, $config['exporter']);
         $this->configureWriters($container, $config['writers']);
     }
 
-    private function configureExporter(ContainerBuilder $container, array $config): void
-    {
-        foreach (['csv', 'json', 'xls', 'xml'] as $format) {
-            if (\in_array($format, $config['default_writers'], true)) {
```

Comment:
But this tag only gets added if this condition is met, you are adding it always.

Reply:
🤔 sounds a little bit weird, tags should be always added IMO, but looks like `Exporter::getAvailableFormats()` is based on these tags being added instead of the configuration parameter.

Anyway, I've just change the configuration to PHP.